### PR TITLE
Update burn-in Test

### PR DESCRIPTION
**Pull request by ctierneynv**
Rename burn-in test to bit, update hpl code to use containers, restructure code for future additional tests.

### CODE REVIEW

File: workloads/burn-in/README.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+The HPL burnin test will repeatedly run HPL on different node counts to verify that each node, and each group of nodes, is providing the expected performance.  If a job runs slowly, it is an indicator that there is an issue with the node or network.  All nodes should perform equally.  
 
-Currently, only clusters built with DGX-1V-16GB, DGX-1V-32GB, DGX-2, and DGXA100 are supported.  If you have an OEM GPU-based system, contact your NVIDIA SA for additional assistance.
+If the expected performance is seen, the user can be confident that the nodes are working correctly.
+
+
+### NCCL Overview
+(This test is still a work in progress)
+
+The NCCL test will use the NCCL all_reduce_perf test to validate fabric correctness and performance.  Testing has only been done on InfiniBand based fabrics.
```
Comment (supertetelman) [inline workloads/burn-in/README.md:15]: Extra space.

File: workloads/burn-in/README.md [hunk 2]
```diff
@@ -44,24 +56,71 @@ NOTE: For the Burn In Test, select the number of jobs (--count ) as 1 to run sin
...
-NITERS: 5
-CUDA_VER: 10.1
-PART: batch
-SET_HCA_AFF: 0
-MAXNODES: 1
-MPIOPTS: 
-TOTAL_NODES: 1
-GRESSTR: --gpus-per-node=8
-...
+HPL_DIR: /home/juser/deepops/workloads/burn-in
```
Comment (supertetelman) [inline workloads/burn-in/README.md:64]: Is this juser path something we want to expose/document here?

File: workloads/burn-in/launch_hpl_experiment.sh [hunk 3]
```diff
@@ -27,14 +27,16 @@ export HPL_SCRIPTS_DIR=${HPL_SCRIPTS_DIR:-${HPL_DIR}} # The shared directory whe
...
 partition=batch
 usehca=0
 usegres=1
 maxnodes=9999
 mpiopts=""
-walltime=02:00:00
+walltime=00:30:00
 verbose=0
+nores=0
+cruntime=singularity
```
Comment (supertetelman) [inline workloads/burn-in/launch_hpl_experiment.sh:38]: Should this be the enroot default?

File: workloads/burn-in/launch_hpl_experiment.sh [hunk 4]
```diff
@@ -250,9 +385,13 @@ for N in $(seq ${niters}); do
 			# Cleanup experiment
 			#exit $? 
 		fi
+
+                popd
```
Comment (supertetelman) [inline workloads/burn-in/launch_hpl_experiment.sh:389]: Extra space

**Review by supertetelman — APPROVED** [review]
Some minor spelling/formatting questions, but LGTM (untested)

**Comment by ctierneynv** [comment]
Adam,

Enroot is not a supported product and is not the official standard or technology of SuperPOD.   We have no standard and no one will step up and make a decision.  With that said, we can make that the standard.

Craig

From: Adam Tetelman ***@***.***>
Reply-To: NVIDIA/deepops ***@***.***>
Date: Tuesday, March 16, 2021 at 5:43 PM
To: NVIDIA/deepops ***@***.***>
Cc: Craig Tierney ***@***.***>, Author ***@***.***>
Subject: Re: [NVIDIA/deepops] Update burn-in Test (#914)


@supertetelman commented on this pull request.

________________________________

In workloads/burn-in/launch_hpl_experiment.sh<https://github.com/NVIDIA/deepops/pull/914#discussion_r595613935>:

>  verbose=0

+nores=0

+cruntime=singularity

Should this be the enroot default?

**Comment by ctierneynv** [comment]
It’s meant to mean “Generic User”, not that any user has that name.

From: Adam Tetelman ***@***.***>
Reply-To: NVIDIA/deepops ***@***.***>
Date: Tuesday, March 16, 2021 at 5:40 PM
To: NVIDIA/deepops ***@***.***>
Cc: Craig Tierney ***@***.***>, Author ***@***.***>
Subject: Re: [NVIDIA/deepops] Update burn-in Test (#914)


@supertetelman commented on this pull request.

________________________________

In workloads/burn-in/README.md<https://github.com/NVIDIA/deepops/pull/914#discussion_r595612945>:

>

 Experiment Variables:

-EXPDIR: results/2node_dgx1v_16G_20200608104253

-NITERS: 5

-CUDA_VER: 10.1

-PART: batch

-SET_HCA_AFF: 0

-MAXNODES: 1

-MPIOPTS:

-TOTAL_NODES: 1

-GRESSTR: --gpus-per-node=8

-...

+HPL_DIR: /home/juser/deepops/workloads/burn-in

Is this juser path something we want to expose/document here?

**Pull request merged**